ci_test/ansible_lint.sh

```
@@ -0,0 +1,82 @@
+#!/bin/bash -e
+
+# Install Python 3.10 if not already installed
+if ! command -v python3.10 &> /dev/null; then
+    echo "Installing Python 3.10..."
+    apt update
+    apt install -y build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libreadline-dev libffi-dev wget
+    wget https://www.python.org/ftp/python/3.10.13/Python-3.10.13.tgz
+    tar -xf Python-3.10.13.tgz
+    cd Python-3.10.13
+    ./configure --enable-optimizations
+    make -j $(nproc)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)